data/seeds/01-users.js

```diff
@@ -0,0 +1,64 @@
+exports.seed = async knex => {
```

Comment:
It's not bad, but the seeded passwords need to be hashed before saving otherwise we won't be able to login with those users.  Use this format
```
const bcrypt = require("bcryptjs");
const hash = async password => await bcrypt.hash(password, 12);

exports.seed = async function(knex) {
  await knex("users").truncate();
  await knex("users").insert([
    { id: 1, username: "user1", password: `${await hash("asdf")}` },
    { id: 2, username: "user2", password: `${await hash("asdf")}` },
    { id: 3, username: "user3", password: `${await hash("asdf")}` }
  ]);
};
```

Reply:
This will allow you to hash the passwords before saving